tap_harvest/client.py

```diff
@@ -11,8 +12,88 @@
 # Timeout request after 300 seconds
 REQUEST_TIMEOUT = 300
 
+class HarvestError(Exception):
+    pass
+
+class Server5xxError(Exception):
```

Comment:
```suggestion
class Server5xxError(HarvestError):
```

Reply:
Inherited HarvestError instead of Exception.